feat(acc): docs on contracts (#4114)
scaleway/docs-content: console/account/how-to/download-scaleway-contracts.mdx

---
meta:
  title: How to download Scaleway contracts
  description: Download General Terms of Services, Data Processing Agreement, and more.
content:
  h1: How to download Scaleway contracts
  paragraph: Download General Terms of Services, Data Processing Agreement, and more.
tags: account contracts gdpr 
dates:
  validation: 2024-12-12
  posted: 2024-12-12
categories:
  - console
---

> Scaleway contracts define the terms, conditions, and commitments that govern the use of our services. They ensure transparency and compliance with legal and regulatory requirements while outlining the rights and responsibilities of both Scaleway and its customers.

Reviewer: ```suggestion

Scaleway contracts define the terms, conditions, and commitments that govern the use of our services. They ensure transparency and compliance with legal and regulatory requirements while outlining the rights and responsibilities of both Scaleway and its customers.
```

## Contracts available for download 

You can access and download the following types of contracts:

| **Contract type**                     | **Description**                                                                                                                                                                                     |
|---------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|

> | General Terms of Services             | Defines the respective rights and obligations of Scaleway and the client in connection with the use by the client of the services provided by Scaleway.                                             |

Reviewer 2: ```suggestion
| General Terms of Services             | Defines the respective rights and obligations of Scaleway and the client in connection with the client's use of the services provided by Scaleway.                                             |
```

| Specific Conditions                   | The contractual document issued by Scaleway, specific to a service, that defines the conditions of provision and use applicable to said service.                                                    |
| Technical and Organizational Measures | Describes the measures taken by Scaleway to ensure the security of client's data.                                                                                                                   |
| Intrusion Testing Agreement           | Defines the conditions applicable in the context of carrying out intrusion tests by the client or any third party acting on its behalf.                                                             |

## How to download Scaleway contracts

You can download Scaleway contracts either via the [console](https://console.scaleway.com/organization) or Scaleway's website:

### From the Scaleway console
You can access and download the contracts applicable to your Organization:
1. Click the [Contracts](https://console.scaleway.com/organization/contracts) tab in your [Organization dashboard](https://console.scaleway.com/organization).

> 2. Select the contract you want to download.

Reviewer 2: ```suggestion
2. Click the download icon next to the contract you want to download.
```
